lib/Sema/TypeCheckAccess.cpp

```diff
@@ -379,9 +379,9 @@ void AccessControlCheckerBase::checkGenericParamAccess(
     auto diagID = diag::generic_param_usable_from_inline;
     if (downgradeToWarning == DowngradeToWarning::Yes)
       diagID = diag::generic_param_usable_from_inline_warn;
-    auto diag = TC.diagnose(owner,
+    auto diag = TC.diagnose(ownerDecl,
```

Comment:
`ownerDecl->diagnose` please.

Reply:
Actually, this will fall out of the merge conflict resolution...

Reply:
Indeed, I've spotted it now.